integrationtests/as-lucene-directory/src/test/resources/user-provided-infinispan-persistence.xml

```diff
@@ -10,8 +10,12 @@
         <jmx duplicate-domains="true"/>
 
         <replicated-cache name="LuceneIndexesMetadata" mode="SYNC" remote-timeout="25000">
+            <transaction mode="NONE" />
             <persistence passivation="false">
-                <string-keyed-jdbc-store preload="true" key-to-string-mapper="org.infinispan.lucene.LuceneKey2StringMapper" shared="true">
+                <!-- N.B. "shared" is false which is unusual for a JDBC CacheStore,
+                 but for testing purposes we use an in-memory H2 so each WildFly node
+                 will have an independent CacheStore instance -->
+                <string-keyed-jdbc-store preload="true" key-to-string-mapper="org.infinispan.lucene.LuceneKey2StringMapper" shared="false">
```

Comment:
right, I actually do that in some other module but it requires a bit more coding to start only one of these. I don't remember which other module does it ;)